groovy/JenkinsUtils.groovy

```diff
@@ -9,6 +9,15 @@ void runTests(propertyFile) {
   }
 }
 
+void runTestsDocker(propertyFile) {
+  sh "cp devops/f4a/" + propertyFile + " f4a/FitNesseForAppian/fitnesse-automation.properties"
+  sh "docker-compose -f docker/docker-compose.yml up &"
+  sleep(time:10, unit:"SECONDS")
```

Comment:
What is the purpose of this sleep?

Reply:
give the docker-compose time to come up. ideally, the better solution may be to pipe into a file and wait until we see that the "webdrive has started on port 4444", but this i think works until that solution is available

Reply:
This bash script will check for a container built with a selenium image to be up and running. I think this should eliminate the need to write to a file and tail the log. Unless there's a scenario where the container can be up and running without the server actually connecting to the port. 
```suggestion
  while [ -z "$(docker ps -a | awk '{print $1, $2}' | grep selenium |
          awk '{print $1}'  ]; do
    :
  done
```